lib/version.js

```diff
@@ -88,8 +89,82 @@ function setCommand(version, pkg) {
 function set(v, project) {
   v = semver(v.toString());
   project.version(v);
+  setBlipIfOpted(project);
+}
+
+function setBlipIfOpted(project) {
+  var opted = false;
+  var blipScript = false;
+  var scriptPhases = {
+    'preinstall': project.scripts('preinstall'),
+    'install': project.scripts('install'),
+    'postinstall': project.scripts('postinstall')
+  };
+  var availPhases = [];
+  //check if the package opted for blip
+  hasOpted(project, scriptPhases, availPhases, opted, blipScript, function(hasOpted, hasBlipScript) {
+    opted = hasOpted;
+    blipScript = hasBlipScript;
+  });
+
```

Comment:
I haven't always been a fan of early exits, but this would probably be a good place for one:
```js
if (!opted) {
  return;
}
```